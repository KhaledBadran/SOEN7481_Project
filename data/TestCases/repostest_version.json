{
    "functionName": "test_version",
    "className": "TestKMIP",
    "fileName": "/OpenKMIP_&_PyKMIP/kmip_&_tests_&_unit_&_test_kmip.py",
    "projectName": "repos",
    "Label": false,
    "isTest": true,
    "Body": "\"\"\"\n        Verify that the kmip module has a __version__ attribute and that it's\n        value is correct.\n        \"\"\"\nkmip = __import__('kmip')\nself.assertTrue(hasattr(kmip, '__version__'))\nversion = __import__('kmip.version')\nself.assertTrue(hasattr(version, '__version__'))\nobserved = getattr(kmip, '__version__')\nexpected = getattr(version, '__version__')\nself.assertEqual(expected, observed)\n"
}