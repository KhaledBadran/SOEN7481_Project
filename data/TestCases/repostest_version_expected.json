{
    "functionName": "test_version_expected",
    "className": null,
    "fileName": "/EVEprosper_&_ProsperCommon/tests_&_test_prosper_version.py",
    "projectName": "repos",
    "Label": false,
    "isTest": true,
    "Body": "\"\"\"validate __version__ meets expectations\"\"\"\nwith open(VERSION_FILEPATH, 'r') as v_fh:\n    version_from_file = v_fh.read()\nversion_from_file = version_from_file.replace('v', '')\nassert version_from_file == version.__version__\nassert version_from_file == p_version.get_version(PROJECT_HERE_PATH)\n"
}